components/transfer/Transfer.vue

```diff
@@ -300,7 +301,7 @@ export type TargetAddress = {
   usd?: number | string
   token?: number | string
 }
-const isMobile = computed(() => useWindowSize().width.value <= 1024)
+const isMobile = computed(() => useWindowSize().width.value <= 764)
```

Comment:
Why don't you put this part directly inside `TransactionLoader`?

Reply:
Honestly, I was asking myself the same question
I was following the  pattern set out by @Jarsen136  in #6695

since we have two modals(confirm send and loader) that need to be matched  (mobile style or desktop style) it does kinda make sense to control this from the calling component

Reply:
It's OK to me if it matches the design on the mobile